CMakeLists.txt

```diff
@@ -88,4 +88,10 @@ ELSEIF()
   MESSAGE(FATAL_ERROR "Cannot locate edbee-lib editor widget submodule source code, build abandoned!")
 ENDIF()
 
+find_program(CCACHE_FOUND ccache)
```

Comment:
Is this from THIS PR?

Reply:
I added it since I wanted to use it here - but I will revert the commit before merging.

Reply:
👍 Guess it was a useful place to test that feature. 😀